src/components/swap/GasBreakdownTooltip.tsx

```diff
@@ -9,6 +10,8 @@ import { isClassicTrade, isUniswapXTrade } from 'state/routing/utils'
 import styled from 'styled-components/macro'
 import { Divider, ExternalLink, ThemedText } from 'theme'
 
+import { getChainInfo } from '../../constants/chainInfo'
```

Comment:
Import this as `import { getChainInfo } from 'constants/chainInfo'` to match the style of the file.
In general, import using a relative path when the file you're importing from is in the same subtree.

Reply:
done!